types/index.d.ts

```diff
@@ -8,7 +8,7 @@
 /// <reference path="./expressions.d.ts" />
 /// <reference path="./helpers.d.ts" />
 /// <reference path="./middlewares.d.ts" />
-/// <reference path="./indizes.d.ts" />
+/// <reference path="./indeces.d.ts" />
```

Comment:
Another point in favor of "indexes" is that the MongoDB documentation calls them "indexes" rather than "indices": https://www.mongodb.com/docs/manual/indexes/ . I think we should err on the side of consistency with the MongoDB docs on this front